Cargo.toml

```diff
@@ -11,3 +11,4 @@ license = "MIT"
 [dependencies]
 iron = { version = "0.4", default-features = false }
 term = "0.4"
+time = "0.1.35"
```

Comment:
Looks like `std::time` exists now. https://doc.rust-lang.org/std/time. However I'm not sure it did at the time of this PR. :)